xds/internal/xdsclient/authority_test.go

```diff
@@ -1,295 +0,0 @@
-/*
```

Comment:
If this is a pure refactor; can we not keep some of these?

Reply:
Oh those cover the same functionality? The diffs in those e2e tests seemed pretty minor so I thought we just deleted these.

Reply:
The following tests old tests `TestTimerAndWatchStateOnSendCallback`, `TestTimerAndWatchStateOnErrorCallback` and `TestWatchResourceTimerCanRestartOnIgnoredADSRecvError` are now covered in the new test `TestADS_WatchState_StreamBreaks`

And we also have another new test `TestADS_WatchState_TimerFires` which explicitly tests timer firing scenario.